README.md

```diff
@@ -1,27 +1,137 @@
-# `ortac` - OCaml Runtime Assertion Checking.
+# `ortac` — OCaml RunTime Assertion Checking
+
+`ortac` is a tool to convert an OCaml module interface with [Gospel]
+specifications into code to check those specifications.
+
+[Gospel]: https://ocaml-gospel.github.io/gospel/
+
+There are various ways to check specifications:
+
+- the _default_ mode generates a _wrapper_ module, _i.e._ a module
+  exposing the original interface but instrumenting all function calls
+  with assertions on arguments and on results, either normal or
+  exceptional,
+- the _monolith_ mode, provided as a plugin, generates a program using
+  [Monolith] that will test the module by comparing it against an
+  instrumented version of the module; see the [dedicated README] for
+  details.
+
+[Monolith]: https://gitlab.inria.fr/fpottier/monolith
+[dedicated README]: plugins/monolith/README.md
 
-**Disclamer:** This project is still experimental. No support will be provided
-at this point, and its behaviour is still unstable.
 
 ## Installation
 
+This project is still under development (standard warnings apply...),
+with no stable release yet.
+Consequently the easiest way to try it out is to `opam pin` it:
+
 ```
-opam pin add -y https://github.com/ocaml-gospel/gospel.git
 opam pin add -y https://github.com/ocaml-gospel/ortac.git
-opam install ortac
 ```
 
-## how to add a plugin
+It requires a yet-unreleased version of Gospel because Gospel evolved
+quite a lot since its 0.1.0 release.
+
+This repository contains the following OPAM packages:
+
+- `ortac.opam` which provides the `ortac` command-line tool and its
+  default mode,
+- `ortac-runtime.opam` which provides the support library for the code
+  generated by Ortac default mode,
+- `ortac-monolith.opam` which provides the Monolith plugin for the
+  `ortac` command-line tool,
+- `ortac-runtime-monolith.opam` which provides the support library for
+  the code generated with the Monolith plugin.
+
+You can install only some of those packages by explicitly mentionning
+which package you want to install, for instance:
+
+```
+$ opam pin add ortac https://github.com/ocaml-gospel/ortac.git
+```
+
+To check that it is correctly installed, simply run `ortac`. If you
+installed all packages, you should get:
+
+```
+$ ortac
+ortac: required COMMAND name is missing, must be either 'default' or 'monolith'.
```

Comment:
Indeed. But:
- #81 did move the opam packages for the monolith plugin to the root of the repository so that `opam pin add .../ortac.git` would install also the plugin,
- the previous sentence is phrased “If you installed all packages...”.
Do you think it should be rephrased?